src/rust/src/rdataframe/read_ipc.rs

```diff
@@ -15,17 +16,19 @@ pub fn import_arrow_ipc(
     memmap: Robj,
 ) -> Result<RLazyFrame> {
     let args = ScanArgsIpc {
-        n_rows: robj_to!(Option, usize, n_rows)?,
-        cache: robj_to!(bool, cache)?,
-        rechunk: robj_to!(bool, rechunk)?,
-        row_count: robj_to!(Option, String, row_name)?
+        n_rows: robj_to!(Option, usize, n_rows).map_err(|msg| ranyhow!(msg))?,
```

Comment:
If implementing something like `impl<E> From<Result<E, String>> for Result<E, anyhow::Error> {}` then the ? should implicitly call it and map_err would not be needed.

Reply:
Yeah we can use this for transition.